### ISSUE TITLE
Logout does not remove a session that was not started

### ISSUE THREAD

**Issue by dv336699** [allburov/whatsapp-http-api#194]
Engine: NOWEB

Version:

```
{
  "version": "2023.8.29",
  "tier": "PLUS",
  "browser": "/usr/bin/chromium"
}
```

1. Start the container
2. Start the session and you should see the qr code in the terminal (do not scan the qr code)
3. Send a logout command

The expected result is that the session will be removed/stopped, but that doesn't happen and the session keeps open showing the qr code.

**Comment by allburov** [comment]
Is it for noweb engine, right?

**Comment by dv336699** [comment]
@allburov sorry, yes, is there any chance you you can create a template of what information you need so we can send it all in one go?

**Comment by allburov** [comment]
Good idea, I will do! 👍

**Comment by allburov** [comment]
That's correct behaviour - you need to use `/stop` endpoint instead to actual STOP the session with `logout: True`
`/logout` only remove credentials for the session
But I see the confusion here, we'll look at this closer

**Comment by devlikepro** [comment]
Now when you call `/logout` it'll stop the session if it's running. I don't remember exactly version when it's been fixed tho...

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Fixed in `2024.10.9`

When a session in `WORKING` status on `POST /api/sessions/{NAME}/logout`  request it'll try to remove the device from **Connected Devices** list in the app.

https://waha.devlike.pro/docs/how-to/sessions/#logout-session

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
